src/reducers/index.js

```diff
@@ -1,20 +1,21 @@
+import {combineReducers} from "redux";
 
 function currentCount(state=0, action){
   if(action.type === "INCREASE_COUNTER"){
-    
+    return state + 1;
   }
   if(action.type === "DECREASE_COUNTER"){
-    
+    return state - 1;
   }
   return state;
-}   
+}
 
 function users(state =[], action){
   if(action.type === "ADD_USER"){
 
```

Comment:
keep working on it. something should go there. no point in an empty if statement

Reply:
Whoops. thanks for pointing that out